test/common/router/config_impl_test.cc

```diff
@@ -447,6 +447,34 @@ TEST_F(RouteMatcherTest, TestRoutes) {
     route:
       prefix_rewrite: "/api/new_endpoint"
       cluster: www2
+  - match:
+      prefix: "/newforreg1_endpoint"
+    route:
+      regex_rewrite:
+        pattern:
+          google_re2: {}
+          regex: "^/new(.*?)_endpoint(.*)$"
+        substitution: "/\\1_rewritten_endpoint\\2"
```

Comment:
I think you can drop the double quotes and then shouldn't have to escape `\`, which might make these configs easier to read.

Reply:
Changed as suggested.